### PR TITLE
Avoid UI conflict between odf-console and odf-multicluster-console using flags

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

File: console-extensions.json [hunk 1]
```diff
@@ -15,6 +15,17 @@
...
   },
+  {
+    "type": "console.flag/model",
+    "properties": {
+      "model": {
+        "group": "ramendr.openshift.io",
+        "version": "v1alpha1",
+        "kind": "DRPolicy"
+      },
+      "flag": "ODR_MODEL"
```
Comment (bipuladh) [inline console-extensions.json:26]: Where is this used?

Comment (GowthamShanmugam) [reply console-extensions.json:26]: For DataService navigation on the ACM page. https://github.com/red-hat-storage/odf-console/pull/183/files#diff-2d79046ce9b0f8fda82dd631ce959a36f6c350960d54cf8fdb1579f3c718e83aR88

Comment (GowthamShanmugam) [reply console-extensions.json:26]: right now we have only one DataPolicies, i.e DisasterRecovery, if the ODR Hub operator is not installed then we dont need to show DataService options.

In future, once we started supporting multiple data services and data policies we will need to hide only the DisasterRecovery tab

Comment (afreen23) [reply console-extensions.json:26]: Is this flag currently defined anywhere ? I cant find that.

**Review by afreen23 — CHANGES_REQUESTED** [review]

File: console-extensions.json (hunk 1 above)
Comment (GowthamShanmugam) [reply console-extensions.json:26]: it is used: https://github.com/red-hat-storage/odf-console/pull/183/files#diff-2d79046ce9b0f8fda82dd631ce959a36f6c350960d54cf8fdb1579f3c718e83aR88

This extension will create this flag based on CRD present

Comment (GowthamShanmugam) [reply console-extensions.json:26]: @afreen23 ^

**Pull request merged**